dts/bindings/iio/adc/nxp,vf610-adc.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2021, Antonio Tessarolo
+# SPDX-License-Identifier: Apache-2.0
+
+description: Vf610 Adc
+
+compatible: "nxp,vf610-adc"
+
+include: adc-controller.yaml
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: true
+
+    "#io-channel-cells":
+      const: 1
+
+    rdc:
+      type: int
+      required: true
+      description: |
+        Set the RDC permission for this peripheral: the RDC controls which
+        processor can access to this peripheral. User can select to assign this
+        pheripheral to the M4 processor, A9 processor or both.
```

Comment:
Thank you for the update, but it's not clear from the description what integer values correspond to what settings. Can you please clarify this?

For example, "Set this to 0 to assign this peripheral to the M4 processor, 1 to assign it to the A9 processor, and 2 for both" or something like that.

You may also want to consider using an `enum` here if there are only a few acceptable values.

Note the "pheripheral" typo as well.